service-statistics-common/src/test/java/fi/nls/oskari/control/statistics/RegionSetHelperTest.java

```diff
@@ -0,0 +1,86 @@
+package fi.nls.oskari.control.statistics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.opengis.geometry.MismatchedDimensionException;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.operation.TransformException;
+import org.oskari.geojson.GeoJSON;
+
+import fi.nls.oskari.control.statistics.db.RegionSet;
+import fi.nls.oskari.service.ServiceException;
+
+public class RegionSetHelperTest {
+
+    @Test
+    @Ignore("Requires HTTP connection to a WFS server serving specific FeatureType")
```

Comment:
I would have liked to keep the input as a resource file as an example of what the regionset config should be for a given WFS-response. But there was some problems with linked resources using the new parser?

Reply:
The used parser (GeoTools GML) is the same as before, but the it won't work when the schemaLocation links etc. in the document don't work. For example current resource files contain links to `dev.paikkatietoikkuna.fi:80`, which won't resolve. This test is also more of an "integration" test in the sense that it's asserting more the fact that the request generated is valid WFS request and that the WFS service responses with a `<wfs:FeatureCollection>` GML document for the request. Parsing the GML is tested more thoroughly within GeoTools already, so that's not the main thing under inspection here.